core/recipe-dsl-kotlin/src/test/kotlin/com/ing/baker/recipe/kotlindsl/KotlinDslTest.kt

```diff
@@ -394,6 +394,8 @@ class KotlinDslTest {
             assertEquals("reservedItems", sieves().get(0).inputIngredients().get(0).name())
             assertEquals(1, sieves().get(0).output().size())
             assertEquals("\$SieveEvent\$extractDate", sieves().get(0).output().get(0).name())
+            assertEquals("1981-01-07", (sieves()[0].function() as Function1<Ingredients.ReservedItems, String>).invoke(
```

Comment:
In the line above, we use a different style to get the first Sieve: `sieves().get(0).output()`, can we change this like `sieves().get(0).function()`?

Reply:
Would you be okay if I changed everythign to the other style. sieves()[0], because that would be the idiomatic way for a kotlin List (and therefore I think should also be the idiomatic way for a Seq.)

Reply:
That's fine for me if this is a more idiomatic way in Kotlin.